### DISCUSSION TITLE
Schedule jobs

### DISCUSSION THREAD

**Discussion by eerison** [okfn-brasil/querido-diario-comunidade#36] [Q&A]
Hello, could you explain a bit for what is used this [scheduler.py](https://github.com/okfn-brasil/querido-diario/blob/main/scripts/scheduler.py) file, And this [Daily execution of Spiders](https://github.com/okfn-brasil/querido-diario/actions/runs/3385173952) is used for what?

**Comment by lucas-0liveira** [comment]
The file scheduler.py is a component of a task scheduling system in Python. It is typically used to automate the execution of specific operations at a given interval or at specific times.

On the other hand, the daily execution of Spiders suggests that you are dealing with a web scraping process. Spiders are scripts or programs that navigate the web, collecting data from websites in an automated fashion.

I hope I've helped.

**Reply by eerison** [reply]
Hey @lucas-0liveira 

Thanks for reply 🙂,  I guess it give me a better idea how it works  ;)

**Reply by lucas-0liveira** [reply]
I'm glad I could help with your question, that's what I'm here for!
